### PR TITLE
8105 - Explicitly enable EXT_float_blend to silence WebGL warnings

### LINKED ISSUE #8105: ol-cesium chrome 76: too many errors
After chrome update:
```
256[.WebGL-0x2588fcac5300]GL ERROR :GL_INVALID_OPERATION : glDrawElements: GL_BLEND with floating-point color attachments requires the EXT_float_blend extension
7796ee5f-aab2-46bf-8f30-340669acc40a:1 WebGL: too many errors, no more errors will be reported to the console for this context.
```
And no features on screen.
Before Chrome 76 everything works fine.
Cesium `1.60.0`
Is it chrome, ol-cesium or cesium bug?

### PR DESCRIPTION
**Pull request by Samulus**
Fixes #8105 

You can test this PR by:

1) Opening http://localhost:8080/Apps/Sandcastle/index.html?src=Billboards.html in Firefox and observing `Error: WebGL warning: drawElementsInstanced: Using format enabled by implicitly enabled extension: EXT_float_blend. For maximal portability enable it explicitly. Context.js:321:37` is printed to the console.
2) Building with this PR and verifying this message is no longer logged. 

I'm using Firefox 72.0.1 on Windows 10.0.18363.418, Couldn't reproduce warning message in Chrome 79.0.3945.117

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @Samulus! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/AnalyticalGraphicsInc/cesium/blob/8105/EXT_float_blend/CONTRIBUTORS.md) file!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/8105/EXT_float_blend/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: CHANGES.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Change Log
 
 ##### Fixes :wrench:
 * Fixed a bug where the camera could go underground during mouse navigation. [#8504](https://github.com/AnalyticalGraphicsInc/cesium/pull/8504)
+* Fixed WebGL warning message about `EXT_float_blend` being implicitly enabled
```
Comment (lilleyse) [inline CHANGES.md]:
```suggestion
* Fixed WebGL warning message about `EXT_float_blend` being implicitly enabled. [#8534](https://github.com/AnalyticalGraphicsInc/cesium/pull/8534)
```

**Comment by lilleyse** [comment]
@Samulus Works perfectly. Just one small fix needed for `CHANGES.md`. You can put `[skip ci]` in the commit message or description to avoid starting another CI build.

**Comment by lilleyse** [comment]
Confirmed that this fix works in Firefox 72.0.1 and Ubuntu 18.04.2.

**Pull request merged**